annotator/openpose/body.py

```diff
@@ -7,10 +7,23 @@
 import matplotlib
 import torch
 from torchvision import transforms
+from typing import NamedTuple, List
 
 from . import util
 from .model import bodypose_model
 
+class Keypoint(NamedTuple):
+    x: float
+    y: float
+    score: float = 1.0
```

Comment:
`score` and `id` are not used anywhere else in the code. Do we want to keep them? Is `score` here the same concept of `confidence`?